src/backend/api/Fusion.Resources.Domain/Services/ProfileServices.cs

```diff
@@ -35,20 +35,25 @@ public ProfileServices(IFusionProfileResolver profileResolver, ResourcesDbContex
             return existingEntry;
         }
 
-        public async Task<DbExternalPersonnelPerson?> RefreshExternalPersonnelAsync(PersonId personId)
+        public async Task<DbExternalPersonnelPerson> RefreshExternalPersonnelAsync(PersonId personId)
         {
             var profile = await ResolveProfileAsync(personId);
             var resolvedPerson = await ResolveExternalPersonnelAsync(personId);
 
             //in the unlikely event of non-existent person, add it to database
```

Comment:
Did not catch this the last round, but, in what scenario are we refreshing a profile that is not in the database? Should we not just throw on that? If it's not in the database, there is no need to get any latest changes...?

Reply:
Good point, will change this.